### PR TITLE
Created IAM plugin for password policy exists

### PR DESCRIPTION
**Pull request by aqibbutt20**

### CODE REVIEW

File: plugins/aws/iam/passwordPolicyExists.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Password Policy Exists',
+    category: 'IAM',
+    domain: 'Identity and Access management',
+    description: 'Ensures that password policy Enabled',
```
Comment (alphadev4) [inline plugins/aws/iam/passwordPolicyExists.js:7]:
```suggestion
    description: 'Ensures that password policy is set for IAM user.',
```

Comment (alphadev4) [reply plugins/aws/iam/passwordPolicyExists.js:7]: password policy is set for each IAM user

Comment (alphadev4) [reply plugins/aws/iam/passwordPolicyExists.js:7]: password policy is set for each IAM user

**Review by alphadev4 — APPROVED** [review]

File: plugins/aws/iam/passwordPolicyExists.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Password Policy Exists',
+    category: 'IAM',
+    domain: 'Identity and Access management',
+    description: 'Ensures that password policy is set for IAM users.',
+    more_info: 'You can set a custom password policy on your AWS account to specify complexity requirements and mandatory rotation periods for your IAM users\' password.',
```
Comment (AkhtarAmir) [inline plugins/aws/iam/passwordPolicyExists.js:8]:
```suggestion
    more_info: 'You can set a custom password policy on your AWS account to specify login password complexity requirements and mandatory rotation periods for your IAM users\' passwords.',
```

File: plugins/aws/iam/passwordPolicyExists.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Password Policy Exists',
+    category: 'IAM',
+    domain: 'Identity and Access management',
```
Comment (AkhtarAmir) [inline plugins/aws/iam/passwordPolicyExists.js:6]:
```suggestion
    domain: 'Identity and Access Management',
```

File: plugins/aws/iam/passwordPolicyExists.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            helpers.addResult(results, 2, 'Account does not have a password policy');
+            return callback(null, results, source);
+        }
+
+        if (getAccountPasswordPolicy.err || !getAccountPasswordPolicy.data) {
+            helpers.addResult(results, 3,
+                'Unable to query for password policy status: ' + helpers.addError(getAccountPasswordPolicy));
+            return callback(null, results, source);
+        }
+        helpers.addResult(results, 0, 'Account has a password policy');
```
Comment (AkhtarAmir) [inline plugins/aws/iam/passwordPolicyExists.js:36]:
```suggestion
        }

        helpers.addResult(results, 0, 'Account has a password policy');
```

**Review by AkhtarAmir — APPROVED** [review]

**Pull request merged**